TrainingApp/Controller/SettingsViewController.swift

```diff
@@ -119,6 +174,87 @@ final class SettingsViewController: UIViewController {
     
     // MARK: - Helpers
     
+    private func checkThemeSwitchState() {
+        if(userDefaults.bool(forKey: ON_OFF_KEY_THEME)) {
+            darkModeSwitch.setOn(true, animated: false)
+            self.userDefaults.set(self.DARK_THEME, forKey: self.THEME_KEY)
+        }
+        else {
```

Comment:
remove this space
make it 
if {
} else {
}

Reply:
Done 👍